batchflow/models/torch/lightning.py

```diff
@@ -0,0 +1,86 @@
+""" Utilities to seamlessly convert `BatchFlow` model and data generation pipeline to
+`Pytorch Lightning` compatible modules to leverage all of its features.
+
+>>> lightning_model = batchflow_model.to_lightning()
```

Comment:
There is still no way to convert a batchflow model to a custom lightning module class.